packages/codemods/src/command.ts

```diff
@@ -1,18 +1,59 @@
-export class CommandParserResult {
-  // path: string = ""; TODO this doesn't work with jest; why?
-  constructor(public shouldExit = false, public path = '') {}
+import yarr from 'yargs';
+import { Maybe } from './maybe';
+import { CodeMod } from './codeMods/types';
+import { getModFilter, getModExcludeFilter } from './modRunner/modFilter';
+export interface CommandParserResult<T = CodeMod> {
+  shouldExit?: boolean;
+  modsFilter: (mod: T) => boolean;
+}
+export interface Commands {
+  help?: boolean;
+  filter?: string;
 }
 
-export class CommandParser {
-  constructor(private logger = console.log) {}
-
-  public parseArgs(argv: string[]) {
-    if (argv.slice(2).indexOf('--help') >= 0) {
-      this.logger(['Usage:', [argv[0], argv[1], '<path in project>'].join(' ')].join('\n'));
+export const yargsParse = (passedArgs: string[]) => {
+  return yarr([])
+    .help()
+    .exitProcess(false)
+    .option('modNames', {
+      alias: 'n',
+      type: 'string',
+      array: true,
+      description: 'A list of strings of mod names to run',
+    })
+    .option('modNameRegexs', {
```

Comment:
I'm pretty sure this should be `modNameRegexes`? Or to make it easier to type just `modRegexes`. Or `modPatterns` to get around the issue of how to pluralize "regex"? ("pattern" is commonly used in regex options elsewhere)